### PR TITLE
Vp 1634 update onboarding school ready video text

### PR DESCRIPTION
**Pull request by Eteroa123**
## I do solemnly swear that I have:
- [*] Run `npm test` and all the tests pass.
- [*] Run `npm run lint` and there are no warnings.
- [*] Not decreased the overall test coverage?
- [*] Included the Jira ID and description in the PR title

## Proposed Changes? 🤔
1. Changed the video from the broken one
2. Added new questions
3. Updated the test 

## Screenshots 📷
 
Screen Shot 2020-04-07 at 12 52 56 AM

### CODE REVIEW

File: components/quiz/__tests__/quiz.fixture.js [hunk 1]
```diff
@@ -1,33 +1,56 @@
...
+    name: 'Are you Volunteer Ready?',
     description: 'Watch this video to learn what you need to know when working in a school',
     badgeclass: 'Gp99Py5ERQGVeDFj63gizA',
-    src: 'https://www.youtube.com/embed/kv8GrLpfVNo',
+    src: 'https://www.youtube.com/embed/N0Ktkx8SYH0',
     questions: [
       {
-        name: 'tShirt',
-        q: 'What does Vaughan have on his t-shirt?',
+        name: 'Tama',
```
Comment (avowkind) [inline components/quiz/__tests__/quiz.fixture.js:9]: not a good idea to put the answer in the question name - change this to name

File: components/quiz/__tests__/quiz.fixture.js [hunk 2]
```diff
@@ -1,33 +1,56 @@
...
-          '2022'
+          'Remote Working Policies',
+          'Delivering Medication',
+          'Groceries Delivery',
+          'Redux Tutorial'
+        ]
+      },
+      {
+        name: 'Sharon',
+        q: 'What is the name of the Requestor?',
```
Comment (avowkind) [inline components/quiz/__tests__/quiz.fixture.js:30]: what is the name of the person asking for help

File: pages/goal/volunteer/ready.js [hunk 3]
```diff
@@ -8,32 +8,55 @@ const readyqs = [
   {
     name: 'Are you ready to volunteer?',
     description: 'Watch this video to learn what you need to know when volunteering with Voluntarily.',
```
Comment (avowkind) [inline pages/goal/volunteer/ready.js:10]: stop after - When volunteering.    leave off with Voluntarily

File: pages/goal/volunteer/ready.js [hunk 4]
```diff
@@ -8,32 +8,55 @@ const readyqs = [
...
-    src: 'https://www.youtube.com/embed/kv8GrLpfVNo',
+    src: 'https://www.youtube.com/embed/N0Ktkx8SYH0',
     badgeclass: 'Gp99Py5ERQGVeDFj63gizA',
```
Comment (avowkind) [inline pages/goal/volunteer/ready.js:12]: both videos have the same badge class - we need to create a new badge.

Comment (Eteroa123) [reply pages/goal/volunteer/ready.js:12]: Is there a way to issue a new badge class ? @avowkind

**Pull request merged**